dask_cuda/tests/test_dgx.py

```diff
@@ -272,11 +277,16 @@ def _test_dask_cuda_worker_ucx_net_devices(enable_rdmacm):
 
                 start = time()
                 while True:
-                    if len(client.scheduler_info()["workers"]) == get_gpu_count():
+                    n_gpus = get_n_gpus()
+                    if len(client.scheduler_info()["workers"]) == n_gpus:
                         break
-                    else:
-                        assert time() - start < 10
-                        sleep(0.1)
+                    elif time() - start > 2 * n_gpus:
```

Comment:
How did we determine this amount of time?

Reply:
Trial and error, this seems to be enough for a DGX-2, but the previous 10 seconds that was hardcoded wasn't.